src/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -156,6 +245,113 @@ public abstract partial class JsonNamingPolicy
         public static System.Text.Json.JsonNamingPolicy CamelCase { get { throw null; } }
         public abstract string ConvertName(string name);
     }
+    public partial class JsonNode
+    {
+        public JsonNode() { }
+    }
+    public partial class JsonNull : System.Text.Json.JsonNode
+    {
+        public JsonNull() { }
```

Comment:
Do we want to have a constant here similar to what we have in string.Empty? Does it even matter?